libc/src/__support/OSUtil/linux/exit.cpp

```diff
@@ -10,19 +10,18 @@
 #include "syscall.h"     // For internal syscall function.
 #include <sys/syscall.h> // For syscall numbers.
 
-namespace LIBC_NAMESPACE {
+namespace LIBC_NAMESPACE::internal {
 
 // mark as no_stack_protector for x86 since TLS can be torn down before calling
 // quick_exit so that the stack protector canary cannot be loaded.
 #ifdef LIBC_TARGET_ARCH_IS_X86
 __attribute__((no_stack_protector))
 #endif
 __attribute__((noreturn)) void
```

Comment:
This one probably should be changed to `[[noreturn]]` to match others.

Reply:
I think there was an issue with `[[]]` notation for multiple attributes, IIRC.

Reply:
keeping ((noreturn)) the same for now.

Reply:
https://godbolt.org/z/rhYeP3vjM which part doesn't work?

Reply:
ah, then perhaps I misremember. Sorry for not just checking it myself!

Reply:
should this just be attribute[[noreturn]] then?

Reply:
ah, GCC doesn't recognize the `[[clang::` prefix, that was the issue! Build is now broken from https://github.com/llvm/llvm-project/pull/93620

Reply:
using `[[gnu::` instead should work for gcc: https://godbolt.org/z/6dbrc6cY6

Reply:
Clang doesn't recognize [[gnu::

Reply:
hmm, interesting, turns out it is supported since clang 18: https://godbolt.org/z/G1srfsWen

Reply:
I should have put a comment there. 🤣

This is the third time ppl get confused.

Reply:
> hmm, interesting, turns out it is supported since clang 18: https://godbolt.org/z/G1srfsWen

Because we filed a issue just for this place.